tests/spdx/parser/rdf/test_file_parser.py

```diff
@@ -35,3 +37,18 @@ def test_parse_file():
     assert file.license_comment == "licenseComment"
     assert file.notice == "fileNotice"
     assert file.attribution_texts == ["fileAttributionText"]
+
+
+def test_parse_invalid_file():
+    graph = Graph().parse(os.path.join(os.path.dirname(__file__), "data/invalid_documents/file_without_spdx_ids.xml"))
+    file_node = graph.value(predicate=RDF.type, object=SPDX_NAMESPACE.File)
+    doc_namespace = "https://some.namespace"
+
+    assert isinstance(file_node, BNode)
+    with pytest.raises(SPDXParsingError) as err:
+        _ = parse_file(file_node, graph, doc_namespace)
```

Comment:
```suggestion
        parse_file(file_node, graph, doc_namespace)
```
Also at the other locations.

Reply:
fixed